sapi/fpm/php-fpm.conf.in

```diff
@@ -125,6 +125,14 @@
 ; Default value: 10
 ;systemd_interval = 10
 
+; Bind the master process to a cpu set.
+; The value can be one cpu id, a range or a list thereof.
+;
+; Default Value: not set
+; process.cpumask = "4"
+; process.cpumask = "0-3"
+; process.cpumask = "1-4;8-16;32-48"
```

Comment:
you re not totally wrong even tough I did on purpose to be ressemblant towards Linux's taskset/FreeBSD cpuset's "mindset".

Reply:
Still not sure if it's a good idea but if we want to use this, it would be better to call it `process.cpu_list`. Also why did you choose semicolons - taskset uses commas, right?